dandi/dandiapi.py

```diff
@@ -485,6 +620,7 @@ def _get_data(self) -> Dict[str, Any]:
 
     @property
     def version_id(self) -> str:
+        """The identifier for the version in question of the Dandiset"""
```

Comment:
```suggestion
        """The identifier for the Dandiset version"""
```
since they're is no question/parameter and other methods don't use such a form